SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -1241,24 +1074,4 @@ Tpm2PcrReadForActiveBank (
   OUT     TPML_DIGEST  *HashList
   );
 
-// MU_CHANGE [BEGIN]

```

Comment:
Because we are changing a MU_CHANGE, it may be cleaner to revert that MU_CHANGE and apply it cleanly on top of your lib changes (reason being we can't take the other MU_CHANGE to edk2 as part of this, it would have to be a separate thing, so then if the help lib gets checked in, we'd have to do the cleanup w.r.t. the old MU_CHANGE anyway